mesh/ncmesh.hpp

```diff
@@ -68,7 +68,20 @@ struct CoarseFineTransformations
                            Table &coarse_to_fine,
                            Array<int> &coarse_to_ref_type,
                            Table &ref_type_to_matrix,
-                           Array<Geometry::Type> &ref_type_to_geom) const;
+                           Array<Geometry::Type> &ref_type_to_geom,
+                           bool get_coarse_to_fine_only = false) const;
+
+   void GetCoarseToFineMap(const Mesh &fine_mesh,
+                           Table &coarse_to_fine) const
+   {
```

Comment:
Would it be possible to move the implementation to the `cpp` file to keep the header concise?

Reply:
Done. Thanks @jakubcerveny